JVAlertController/JVAlertController/JVPopoverPresentationController.m

```diff
@@ -26,14 +26,92 @@
 #import <objc/runtime.h>
 #import <UIKit/UIKit.h>
 
-@interface JVPopoverPresentationController : NSObject
+@interface JVPopoverPresentationController : NSObject <UIPopoverControllerDelegate>
 @property (nonatomic, strong) UIView *sourceView;
 @property (nonatomic) CGRect sourceRect;
 @property (nonatomic, strong) UIBarButtonItem *barButtonItem;
+@property (nonatomic, weak) id<UIPopoverPresentationControllerDelegate> delegate;
+@property(nonatomic, readwrite) UIEdgeInsets popoverLayoutMargins;
+@property(nonatomic, copy) UIColor *backgroundColor;
+@property(nonatomic, copy) NSArray *passthroughViews;
+@property(nonatomic, readwrite, strong) Class< UIPopoverBackgroundViewMethods > popoverBackgroundViewClass;
+@property(nonatomic, assign) UIPopoverArrowDirection permittedArrowDirections;
+@property(nonatomic, readonly) UIPopoverArrowDirection arrowDirection;
+@property(nonatomic, readwrite, weak) UIPopoverController *jv_legacyPopoverController;
 @end
 
 @implementation JVPopoverPresentationController
 
+@dynamic popoverLayoutMargins, backgroundColor, passthroughViews, popoverBackgroundViewClass, arrowDirection;
+
+- (UIEdgeInsets)getPopoverLayoutMargins {
+    return self.jv_legacyPopoverController.popoverLayoutMargins;
+}
```

Comment:
Are the getters intentionally prefixed with "get"? I would think you just want ````popoverLayoutMargins````. Same for the other getters below.